README.md

```diff
@@ -65,30 +65,30 @@ Please see the [Demo Docker Repository](https://hub.docker.com/r/rapidsai/rapids
 
 cuDF can be installed with conda ([miniconda](https://conda.io/miniconda.html), or the full [Anaconda distribution](https://www.anaconda.com/download)) from the `rapidsai` channel:
 
-For `cudf version == 21.08` :
+For `cudf version == 22.06` :
 ```bash
 # for CUDA 11.0
 conda install -c rapidsai -c nvidia -c numba -c conda-forge \
-    cudf=21.08 python=3.7 cudatoolkit=11.0
+    cudf=21.08 python=3.8 cudatoolkit=11.0
```

Comment:
Since we officially support 3.9, can we future-proof this a bit more and go straight to 3.9?